CriticalMass/MapViewController.swift

```diff
@@ -83,12 +85,43 @@ class MapViewController: UIViewController {
         gpsDisabledOverlayView.set(title: String.mapLayerInfoTitle, message: String.mapLayerInfo)
         gpsDisabledOverlayView.addButtonTarget(self, action: #selector(didTapGPSDisabledOverlayButton))
         view.addSubview(gpsDisabledOverlayView)
+        gpsDisabledOverlayView.addLayoutsSameSizeAndOrigin(in: view)
+        updateGPSDisabledOverlayVisibility()
+    }
+
+    public func presentMapInfo(with configuration: MapInfoView.Configuration) {
+        dismissMapInfo()
+
+        let view = MapInfoView.fromNib()
+        view.configure(with: configuration)
+        self.view.addSubview(view)
+
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
would move `translatesAutoresizingMaskIntoConstraints` under to the `view` creation